packages/application-generic/src/usecases/compile-template/compile-template.usecase.ts

```diff
@@ -62,7 +63,32 @@ Handlebars.registerHelper(
         }
       })
       .filter((value, index, self) => self.indexOf(value) === index);
-  }
+  },
+);
+
+Handlebars.registerHelper(
+  HandlebarHelpersEnum.GROUPBY,
```

Comment:
```suggestion
  HandlebarHelpersEnum.GROUP_BY,
```
As per my suggestion.